runtime/near-vm-logic/src/logic.rs

```diff
@@ -609,6 +609,37 @@ impl<'a> VMLogic<'a> {
         Ok(self.context.epoch_height)
     }
 
+    /// Returns the stake of an account, if the account is currently a validator. Otherwise returns 0.
+    ///
+    /// # Cost
+    ///
+    /// For not nul-terminated account id:
+    /// `base + read_memory_base + read_memory_byte * num_bytes + utf8_decoding_base + utf8_decoding_byte * num_bytes + memory_write_base + memory_write_size * 16`
+    ///
+    /// For nul-terminated account id :
+    /// `base + (read_memory_base + read_memory_byte) * num_bytes + utf8_decoding_base + utf8_decoding_byte * num_bytes + memory_write_base + memory_write_size * 16`
+    pub fn validator_stake(
```

Comment:
(Just want to clarify) Adding new methods and variables to the VMLogic makes the newly compiled contracts incompatible with an old chain, right?

Reply:
Sorry about that. For some reason I thought I have already bumped protocol version.